Content/Infrastructure/Doctrine/RouteRemover.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of Sulu.
+ *
+ * (c) Sulu GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Sulu\Bundle\ContentBundle\Content\Infrastructure\Doctrine;
+
+use Doctrine\Common\EventSubscriber;
+use Doctrine\Common\Util\ClassUtils;
+use Doctrine\ORM\Event\LifecycleEventArgs;
+use Doctrine\ORM\Events;
+use Sulu\Bundle\ContentBundle\Content\Domain\Model\ContentRichEntityInterface;
+use Sulu\Bundle\RouteBundle\Entity\RouteRepositoryInterface;
+
+class RouteRemover implements EventSubscriber
```

Comment:
This should not be a public service so it should be internal and final:

```suggestion
/**
 * @internal
 * @final
 */
class RouteRemover implements EventSubscriber
```

Reply:
done